src/qiskit_iqm/fake_backends/quantum_architectures/adonis.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2022 Qiskit on IQM developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""File to represent IQMs 5-qubit architecture
```

Comment:
```suggestion
"""Adonis is IQM's 5-qubit architecture with star topology: the central qubit is connected to the other four
```

Reply:
Could also copy the ASCII drawing of the qubit layout from https://github.com/iqm-finland/cirq-on-iqm/blob/main/src/cirq_iqm/devices/adonis.py